tests/test_workers/test_tasks/test_rubygems.py

```diff
@@ -57,11 +62,15 @@ def test_get_config_file(tmp_path, exists):
         )
         with pytest.raises(CachitoError, match=msg):
             rubygems._get_config_file_for_given_package(
-                pkg_and_deps_info["dependencies"], bundle_dir, package_root, rubygems_hosted_repo
+                pkg_and_deps_info["dependencies"],
+                bundle_dir,
+                package_root,
+                rubygems_hosted_repo,
+                None,
```

Comment:
I would appreciate an extra unit test where the subpath is not None, but since one of the cases in `test_fetch_rubygems_source` already tests that it's not critical